src/main/java/greencity/security/repository/OwnSecurityRepo.java

```diff
@@ -25,4 +26,12 @@
     @Modifying
     @Query("update OwnSecurity o set o.password = :password where o.user.id = :id")
     void updatePassword(@Param("password") String password, @Param("id") Long id);
+
+    /**
+     * Method for finding user password.
+     *
+     * @param user - {@link User}
+     * @return
+     */
+    OwnSecurity findByUser(User user);
```

Comment:
If you have already user entity in service, user has own security! You can get it by using getter for ownSecurity

Reply:
Ok, nice comment, really! i'l fix it.